bdanmaku.lua

```diff
@@ -30,7 +31,8 @@ function download_xml()
 		mp.msg.debug('no XML danmaku found')
 		return
 	end
-	xml_filename = TMPDIR..'/'..os.time()..'.danmaku.xml'
+	os.execute('mkdir -p '..TMPDIR)
+	xml_filename = TMPDIR..mp.get_property('pid')..'.xml'
```

Comment:
I don't want to force people to add a trailing slash at the end. Maybe append a trailing slash to `TMPDIR` unless `TMPDIR:match('/$')`?

Reply:
Update in [51ff623](https://github.com/UlyssesZh/bdanmaku/pull/1/commits/51ff623dabf3fe36318cb9687c7033b1c106b2d1).